### ISSUE TITLE
Not working anymore 

### ISSUE THREAD

**Issue by Kositch** [MartinGris/node-red-contrib-skoda-connect#18]
Error when trying to log-in or fetch data: "{"error":{"errorCode":"RS.security.9007","description":"The context has not gained a legitimation due to missing access rights."}}"

Plugin does not work anymore... Any help? Would love to donate, if someone forked this Node and made it working again and without NR restarts...

**Comment by MHagreis** [comment]
same here, error is: 
{"error":{"errorCode":"RS.security.9007","description":"The context has not gained a legitimation due to missing access rights."}}

last successful connect 29-08-2023 10:36:59 CET

**Comment by Kositch** [comment]
Yeah, seems that plugin is definitely dead :/ Any other options how to connect to Škoda in NR?

**Comment by MartinGris** [comment]
I'm sorry but because I don't drive a Skoda anymore the development on this plugin is freezed.
Everybody is invited to continue the development
